cluster/src/main/java/com/hartwig/pipeline/metadata/SomaticRunMetadata.java

```diff
@@ -29,6 +29,11 @@ static String truncate(final String sample) {
         return sample.length() > MAX_SAMPLE_LENGTH ? sample.substring(0, MAX_SAMPLE_LENGTH) : sample;
     }
 
+    @JsonIgnore
```

Comment:
I think there is also an immutable annotation for this @Default to exclude it from builders, etc.